plugin/someteam.example.com/v1/starlarkmixer/go.mod.go

```diff
@@ -0,0 +1 @@
+package starlarkmixer
```

Comment:
what is this file?

Reply:
A mistake my editor generated. I though I had gotten rid of it, but alas. I'll rebase it out.